index.js

```diff
@@ -1,32 +1,36 @@
 #!/usr/bin/env node
 'use strict';
 
-var fs = require('fs');
-var os = require('os');
-var path = require('path');
-var resolve = require('resolve').sync;
-var spawn = require('child_process').spawn;
-var glob = require('glob');
-var async = require('async');
+const fs = require('fs');
```

Comment:
you changed this file to no longer be executable, is that intended?

Reply:
fixed. PTAL.